### ISSUE TITLE
[dev.icinga.com #13727] Graphite graphs not displayed

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#2668]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/13727

**Created by blacks7 on 2016-12-22 07:43:30 +00:00**

Assignee: _(none)_
Status: _New_ 
Target Version: _(none)_
Last Update: _2016-12-22 07:43:30 +00:00 (in Redmine)_

---

After Upgrading to newest release, graphite permission does not function. "Allow Access to module module graphite (module/graphite)" has no function. Only user with the permission "Allow everthing (\*)" could see the graphite graphs.

All other users only see the Performance data, but no Graphs.

**Attachments**

* [icinga-graphite.PNG](https://dev.icinga.com/attachments/download/3965/icinga-graphite.PNG) blacks7 - _2016-12-22 07:41:17 +00:00_ - _No Graphite_
* [icinga-graphite-perf.PNG](https://dev.icinga.com/attachments/download/3967/icinga-graphite-perf.PNG) blacks7 - _2016-12-22 07:42:23 +00:00_ - _With Graphite_

**Comment by lippserd** [comment]
Duplicates #2686
